test/uk/gov/hmrc/merchandiseinbaggage/pagespecs/pages/SearchGoodsCountryPage.scala

```diff
@@ -46,5 +46,5 @@ class SearchGoodsCountryPage(implicit webDriver: WebDriver) extends DeclarationD
 object SearchGoodsCountryPage {
   def path(idx: Int): String = s"/merchandise-in-baggage/search-goods-country/$idx"
 
-  val title = "In what country did you buy the test good?"
+  val title = "What country are the goods going to? test good?"
```

Comment:
this test can be changed back to the content for Import journey so can just revert changes to this file